Makefile

```diff
@@ -305,9 +305,12 @@ ifeq ($(NATIVE), osx)
   CXXFLAGS += -mmacosx-version-min=$(OSX_MIN)
   LDFLAGS += -mmacosx-version-min=$(OSX_MIN)
   ifdef FRAMEWORK
-    FRAMEWORKSDIR := $(strip $(if $(shell [ -d $(HOME)/Library/Frameworks ] && echo 1), \
+    FRAMEWORKSDIR := $(shell echo $(FRAMEWORKSDIR))
```

Comment:
You are right. https://www.gnu.org/software/make/manual/make.html#Environment this explained it to me